beacon-chain/forkchoice/doubly-linked-tree/store_test.go

```diff
@@ -235,6 +235,7 @@ func TestStore_Prune_NoDanglingBranch(t *testing.T) {
 	s.finalizedCheckpoint.Root = indexToHash(1)
 	require.NoError(t, s.prune(context.Background()))
 	require.Equal(t, len(s.nodeByRoot), 1)
+	require.Equal(t, len(s.nodeByPayload), 1)
```

Comment:
You need to use different payload hashes for this test to make sense

Reply:
@potuz  updated.